pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.builds

```diff
@@ -2,43 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
-  <Import Project="$(MSBuildProjectName).props" />
-
   <ItemGroup>
-    <!-- identity project, runtime specific projects are included by props above. Will not be built on uapaot builds. -->
+    <!-- identity project, runtime specific projects are included uap.rids.props. -->
```

Comment:
I can't tell for sure but didn't we get to the conclusion that because of the Condition bellow the identity package was not getting built on uapaot builds? If so, can we modify the condition as part of this PR so that uapaot vertical builds build the identity package as well.